test/JDBC/input/BABEL-SP_TABLES-vu-verify.sql

```diff
@@ -85,6 +85,18 @@ go
 exec sp_tables @table_name = 'babel_sp_tables_vu_prepare_t_s[o-p]tables2'
 go
 
+exec sp_tables @table_name = 'babel_sp_tables_vu_prepare_t[_]sptables', @table_type = "'TABLE'"
```

Comment:
Let's add tests like this
create table t_s_1(a int)
exec sp_tables @table_name = 'table[_]s[_]1'

create table t_s_2(a int)
exec sp_tables @table_name = 'table[_]s[_]'
exec sp_tables @table_name = 'table[_]s_'

Reply:
Added